applications/callflow/doc/cf_branch_variable.md

```diff
@@ -0,0 +1,70 @@
+### Callflow Branch on a Variable's Value
+
+#### Overview
+
+The `branch_variable` callflow enables you to branch based on value of some field inside one of the a call CCVs, user's document, device's documten or an account's document.
+
+##### Callflow fields
+
+Key | Description | Type | Default | Required
+--- | ----------- | ---- | ------- | --------
+`variable` | Name of variable/property that we are looking for | `string` or a json path | | `true`
+`scope` | the place which the variable should be looked for (`custome_channel_vars`, `device`, `user`, `account`) | `string` | `"cunstom_channel_vars"` | `false`
```

Comment:
@icehess slight typo "custom_channel_vars", also we should have a value for to operate on the kz_endpoint (the merge of device, user and account) no?